azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # # Installing azureml-opendatasets with for some weird reason install Pandas 1.0.0 which is not what we need. So we install back 
</pre><p>Change the wording to something like "will install Pandas 1.0.0, which we'll upgrade to 1.2.4 that's needed for RAPIDS"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>